data/scenes/loudspeakers_1-2.asd

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<asdf>
+  <header>
+    <name>.</name>
+    <description>
+      See the Sec. "Use cases" -> "Multiple Input Multiple Output" for an explanation of what this file does.
```

Comment:
A link would probably be more helpful.

Reply:
Yes! I moved everything to https://github.com/SoundScapeRenderer/listening-test and put a link there.

Reply:
Very nice! See my comments at https://github.com/SoundScapeRenderer/listening-test/pull/1.